sal_interfaces/SALSubsystems.xml

```diff
@@ -556,12 +556,12 @@
 <SALSubsystem>
   <Name>EAS</Name>
   <Description>The CSC for the Environmental Awareness System</Description>
-  <IndexEnumeration>first,second</IndexEnumeration>
+  <IndexEnumeration>no</IndexEnumeration>
   <Generics>yes</Generics>
   <ActiveDevelopers>Wouter van Reeven</ActiveDevelopers>
   <Github></Github>
   <JenkinsTestResults></JenkinsTestResults>
-  <RubinObsContact></RubinObsContact>
+  <RubinObsContact>Wouter van Reeven</RubinObsContact>
   <CSCDocs></CSCDocs>
   <ProductOwner>Brian Stalder</ProductOwner>
   <RelatedDocuments></RelatedDocuments>
```

Comment:
GH review won't let me put this where I want. EAS Simulator and Configuration attributes need to have the appropriate content.

Reply:
I gave those fields appropriate values now.